src/tsickle_compiler_host.ts

```diff
@@ -96,13 +96,16 @@ export class TsickleCompilerHost implements ts.CompilerHost {
   /** externs.js files produced by tsickle, if any. */
   public externs: {[fileName: string]: string} = {};
 
+  private sourceFileToPreexistingSourceMapMap = new Map<ts.SourceFile, SourceMapGenerator>();
```

Comment:
You can drop one "map" suffix here, because it's implied in the type.  (E.g. the other maps below don't have Map at the end.)